features/io.asgardio.java.saml.sdk.server.feature/pom.xml

```diff
@@ -89,7 +89,7 @@
                                 </properties>
                             </adviceFile>
                             <bundles>
-                                <bundleDef>org.wso2.carbon.identity.agent.sso.java:org.wso2.carbon.identity.sso.agent</bundleDef>
+                                <bundleDef>io.asgardio.java.saml.sdk:io.asgardio.java.saml.sdk</bundleDef>
                                 <bundleDef>org.wso2.orbit.org.opensaml:opensaml</bundleDef>
                                 <bundleDef>org.wso2.orbit.joda-time:joda-time</bundleDef>
                                 <bundleDef>org.wso2.orbit.org.openid4java:openid4java</bundleDef>
```

Comment:
Do we need openid4java?

Reply:
Resolved in https://github.com/gayashanbc/identity-agent-sso/pull/2/commits/ccb42095df08cb3685ad47ae534f9dd826d4670e